invenio_files_rest/storage.py

```diff
@@ -28,9 +28,14 @@
 
 import hashlib
 import time
+import tempfile
```

Comment:
So I should move AmazonS3 Class in contrib/s3.py and boto to optional dependcy in setup.py, no problem.